start_cerberus.py

```diff
@@ -492,6 +500,10 @@ def main(cfg):
             record_time(time_tracker)
             pool.close()
             pool.join()
+            if cerberus_publish_status:
+                final_status = get_cerberus_status()
+                logging.info("Final number status" + str(final_status))
+                sys.exit(final_status)
```

Comment:
Think the cerberus exit status should only reflect the code exit status instead of cluster health status given that users will be relying on the go/no-go signal vs cerberus exit code. Thoughts?